src/main/java/at/hannibal2/skyhanni/features/dungeon/m7/DragonInfoUtils.kt

```diff
@@ -0,0 +1,97 @@
+package at.hannibal2.skyhanni.features.dungeon.m7
+
+import at.hannibal2.skyhanni.events.DungeonCompleteEvent
+import at.hannibal2.skyhanni.events.DungeonM7Phase5Start
+import at.hannibal2.skyhanni.events.LorenzWorldChangeEvent
+import at.hannibal2.skyhanni.events.M7DragonChangeEvent
+import at.hannibal2.skyhanni.events.MobEvent
+import at.hannibal2.skyhanni.events.PacketEvent
+import at.hannibal2.skyhanni.utils.ChatUtils
+import at.hannibal2.skyhanni.utils.LocationUtils.isInside
+import at.hannibal2.skyhanni.utils.toLorenzVec
+import net.minecraft.network.play.server.S2APacketParticles
+import net.minecraft.util.EnumParticleTypes
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+class DragonInfoUtils {
+    private var inPhase5 = false
+
+    @SubscribeEvent
+    fun onDragonSpawn(event: MobEvent.Spawn) {
+        if (inPhase5) return
+        if (!event.mob.mobType.isSkyblockMob()) return
+        if (event.mob.name != "Ender Dragon") return
```

Comment:
The name would be "Withered Dragon", if working correctly
```suggestion
        if (event.mob.name != "Withered Dragon") return
```